exercises/linked-list/src/pre_implemented.rs

```diff
@@ -0,0 +1,66 @@
+use {LinkedList, Cursor};
+
+// ============================================================================
```

Comment:
If you use `//!` style, this comment will be applied to the entire module when students run rustdoc.